Makefile

```diff
@@ -1,10 +1,16 @@
 .PHONY: install test docker-test
 
 install:
-	cd `go list -f '{{.Dir}}' github.com/libgit2/git2go` && make install
+	cd `go list -f '{{.Dir}}' github.com/libgit2/git2go` && git submodule update --init && make install && go install
 
 test:
 	go test -ldflags "-extldflags=-L"`go list -f '{{.Dir}}' github.com/libgit2/git2go`/vendor/libgit2/build ./...
 
 docker-test:
 	docker build -t go-vcs . && docker run go-vcs
+dep:
```

Comment:
I think you can remove the `dep` target and modify the README to say "Run `go get ./...` to install dependencies." That way, we don't need to update the `dep` target when we add or remove dependencies.